src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -118,14 +115,14 @@ export default function TokenTable() {
             <TokenDataContainer>
               {tokens.map((token, index) => (
                 <LoadedRow
-                  key={token?.name}
+                  key={token?.address}
                   tokenListIndex={index}
                   tokenListLength={tokens?.length ?? 0}
                   token={token}
-                  ref={tokens.length === index + 1 ? lastTokenRef : undefined}
+                  ref={index + 1 === tokens.length ? lastTokenRef : undefined}
                 />
               ))}
-              {loading && LoadingMoreRows}
```

Comment:
compress the compound boolean render conditional to a computed single boolean:
`const showMoreLoadingRows = Boolean(loading && hasMore)`
...
`{showMoreLoadingRows && LoadingMoreRows}`

Reply:
done